spec.md

```diff
@@ -74,19 +81,22 @@ This specification defines the following terms:
   some action. However, there can also be non-interactive Services that can
   perform the desired actions without direct prompting from the Application.
 
+- *Service Binding*: Represents the request to use a Service Instance. As part
+  of this request there might be a reference to the entity that will use the
+  Service Instance. Service Bindings will often contain the credentials that
+  can then be used to communicate with the Service Instance.
+
 - *Service Broker*: Service Brokers manage the lifecycle of Services. Platforms
   interact with Service Brokers to provision, and manage, Service Instances
   and Service Bindings.
 
 - *Service Instance*: An instantiation of a Service offering.
 
-- *Service Binding*: The representation of an association between an
-  Application and a Service Instance. Often, Service Bindings, will
-  contain the credentials that the Application will use to communicate
-  with the Service Instance.
+- *Service Offerings*: Generic bindable endpoints to extend or tie services
```

Comment:
This one sounds a bit odd since it feels like its trying to define something other than just a normal service.
What about:
```
*Service Offering*: The advertisement of a service that a Service Broker supports.
```
and then , if needed, we could define "Service" too since don't actually do that today.

Reply:
Service is defined.

Reply:
whoa, no idea how i missed that earlier today